src/snowflake/cli/plugins/rpc/manager.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import annotations
+
+import importlib
+import inspect
+import os
+import pkgutil
+from typing import Optional, TypedDict
+
+from pygls.server import LanguageServer
+from snowflake import connector
+from snowflake.cli.api.sql_execution import SqlExecutionMixin
+from snowflake.snowpark import Session
+
+MAX_WORKERS = 5
```

Comment:
This should be configurable from config.toml. We probably will need `[cli.lsp]` section.

Reply:
I can remove it for now to keep it from distracting from the rest of it.